external-scripts.json

```diff
@@ -9,6 +8,5 @@
   "hubot-rules",
   "hubot-scripts-us-federal-holidays",
   "hubot-slack-github-issues",
-  "hubot-slack-reading-list",
```

Comment:
This poor thing never worked.  🙁  https://github.com/mgwalker/hubot-slack-reading-list